packages/ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -593,6 +596,35 @@ export class AngularCompilerPlugin {
     }
   }
 
+  private _warnOnUnusedFiles(compilation: compilation.Compilation) {
+    // Only do the unused TS files checks when under Ivy
+    // since previously we did include unused files in the compilation
+    // See: https://github.com/angular/angular-cli/pull/15030
+    if (!this._compilerOptions.enableIvy) {
+      return;
+    }
+
+    const program = this._getTsProgram();
+    if (!program) {
+      return;
+    }
+
+    // Exclude the following files from unused checks
+    // - ngfactories & ngstyle might not have a corrispond
```

Comment:
Nit: `corrispond` -> `correspondent`

Reply:
I think you marked this as resolved but ended up not changing it.

Reply:
Oops I’ll do it 😊